05-02-2019, SM

# WarmteVerlies Spouwmuur
In dit document worden een aantal verschillende spouwconstructies met elkaar vergeleken voor wat betreft het warmteverlies. 
<br>Oorspronkelijk is dit document/programma ( JupyterLab / Python ) geschreven om het gehele warmteverlies van een bestaande ruimte te modeleren en hiermee te bepalen waar de meeste winst (zowel economisch gezien als ook wat betreft duurzaamheid) valt te behalen.
<br>Hier zoomen we echter in op een aantal spouwmuurconstructies, in een poging de volgende vragen te beantwoorden:
- wat is de winst van het bijvullen van een half gevulde spouw (komt in een groot aantal huizen in Molenhoek voor)
- wat is de winst bij vulling van een geheel lege spouw (klopt de claim van isolatiebedrijven)
- wat is de winst van een extra binnenmuur (dit naar aanleiding van een stel dat zeer enthousiast deze vorm van isolatie in Mook aan het realiseren is)

Een zeer heldere en beknopte uitleg over de functie van een spouw, waarin veel van de mystiek rondom spouwmuren de wereld wordt uitgeholpen, is hier te vinden:
http://www.ekbouwadvies.nl/bouwbesluit/energiezuinigheid/spouwventilatie.asp
<br><br>Bouwbesluit uitgebreide toelichting hoe de Rc-waarde van een gemetselde muur te berekenen
http://www.ekbouwadvies.nl/bouwbesluit/energiezuinigheid/rcwaardebaksteenmuur.asp


## Uitgangspunten

In [8]:
# Maten van de totale Buitenmuur
Lengte  = 9.40
Hoogte  = 2.50

# Veronderstelde temperaturen
Temperatuur_Binnen = 20.0
Temperatuur_Buiten = 0.0

# Voor de berekening van de kostenbesparing gaan we uit van 
# Temperatuurverschil zoals hierboven gedefinieerd
# Dit temperatuurverschil is (gemiddeld) gedurende 100 dagen en 24 uur per dag aanwezig.  DIT MOET MOGELIJK BIJGESTELD WORDEN
# De gasprijs in [Euro] per m3:
Price_Gas = 0.80

## Materiaal Constanten
Hieronder de definitie van de gebruikte materialen.
<br>Iedere vorm om de warmtegeleiding op te geven is toegestaan ( R-waarde, U-waarde of Lambda + Dikte). 

In [9]:
from Base_Heat import *

# luchtweerstand van de binnenmuur, zoals gedefinieerd in het bouwbesluit
Rsi = Heat_Transfer_Object ( "Rsi" )
Rsi.R_Waarde = 0.13

# luchtweerstand van de buitenmuur, zoals gedefinieerd in het bouwbesluit
Rse = Heat_Transfer_Object ( "Rse" )
Rse.R_Waarde = 0.04

# De isolatiewaarde van de spouw tussen de 15 .. 300 mm heeft een vaste R_Waarde
# bron: http://www.ekbouwadvies.nl/bouwbesluit/energiezuinigheid/spouwventilatie.asp
Spouw_Zwak_Geventileerd = Heat_Transfer_Object ( "Spouw Zwak Geventileerd" )
Spouw_Zwak_Geventileerd.R_Waarde = 0.18   

# Veel gebruikte steen voor binnenmuren
Kalkzandsteen = Heat_Transfer_Object ( "Kalkzandsteen" )
Kalkzandsteen.R_Waarde = 0.1

#Veel gebruikte steen voor buitenmuren
Baksteen = Heat_Transfer_Object ( "Baksteen" )
Baksteen.R_Waarde = 0.08

Glaswol_5cm = Heat_Transfer_Object ( "Glaswol 5cm" )
Glaswol_5cm.Lambda = 0.035
Glaswol_5cm.Dikte  = 0.05

Iso_Parels_5cm = Heat_Transfer_Object ( "Iso-Parels 5cm" )
Iso_Parels_5cm.Lambda = 0.034
Iso_Parels_5cm.Dikte  = 0.05

PIR_alu_10cm = Heat_Transfer_Object ( "PIR_alu_10cm" )
PIR_alu_10cm.R_Waarde = 4.54

Leem_Deklaag = Heat_Transfer_Object ( "Leem_Deklaag" )
Leem_Deklaag.Lambda = 1.2
Leem_Deklaag.Dikte  = 0.01

Gipsplaat = Heat_Transfer_Object ( "Gipsplaat" )
Gipsplaat.R_Waarde = 0.04

print ( PIR_alu_10cm )

Name        = PIR_alu_10cm
R-Waarde    = 4.54 [m2.K/W]
U-Waarde    = 0.22 [W/m2.K]



## Muur met half gevulde spouw
Dit is een model van mijn eigen muur, waarmee het allemaal begon. Een 10 cm spouw, die half gevuld is met isover glaswol.
<br>Het isolatiebedrijf gaf na 1 endoscopisch blik in de muur aan, dat het de muur niet wilde bij-isoleren. Onduidelijk is gebleven wat daarvan de reden was, ik meen zoiets als "condensatie" gehoord te hebben, maar dat is natuurlijk faillikante onzin.

In [10]:
Buitenmuur_Nu = Sandwich ( "Buitenmuur Nu" )
Buitenmuur_Nu.Inpandig            = False
Buitenmuur_Nu.Lengte              = Lengte
Buitenmuur_Nu.Hoogte              = Hoogte
Buitenmuur_Nu.Temperature_Inside  = Temperatuur_Binnen
Buitenmuur_Nu.Temperature_Outside = Temperatuur_Buiten
Buitenmuur_Nu.Add ( Rsi )
Buitenmuur_Nu.Add ( Kalkzandsteen )
Buitenmuur_Nu.Add ( Glaswol_5cm )
Buitenmuur_Nu.Add ( Spouw_Zwak_Geventileerd )
Buitenmuur_Nu.Add ( Baksteen )
Buitenmuur_Nu.Add ( Rse )

print ( Buitenmuur_Nu )

Name        = Buitenmuur Nu
Lengte      = 9.4 [m]
Hoogte      = 2.5 [m]
Oppervlakte = 23.5 [m2]
R-Waarde    = 1.96 [m2.K/W]
U-Waarde    = 0.51 [W/m2.K]
T-Inside    = 20.0 [Celcius]
T-Outside   = 0.0 [Celcius]
Heat Loss   = 239 [W]
===========  Kosten per m2, 24 uur per dag, 100 dagen =========
Heat Loss 1m2   = 10 [W]
Energy Cost 1m2 = 1.96 [Euro]
Gasprijs        = 0.80 [Euro]
Layer[s]    = [ Rsi, Kalkzandsteen, Glaswol 5cm, Spouw Zwak Geventileerd, Baksteen, Rse,  ]
Dikte[s]    = [ -1.000, -1.000, 0.050, -1.000, -1.000, -1.000,  ] [m]
R-Waarde[s] = [ 0.13, 0.10, 1.43, 0.18, 0.08, 0.04,  ] [m2.K/W]
U-Waarde[s] = [ 7.69, 10.00, 0.70, 5.56, 12.50, 25.00,  ] [W/m2.K]
T-Inside    = 20.0 [Celcius]
T-Outside   = 0.0 [Celcius]



## Muur met half gevulde spouw + bijgevuld

In [11]:
Buitenmuur_Na = Sandwich ( "Buitenmuur Na Bijvulling" )
Buitenmuur_Na.Inpandig            = False
Buitenmuur_Na.Lengte              = Lengte
Buitenmuur_Na.Hoogte              = Hoogte
Buitenmuur_Na.Temperature_Inside  = Temperatuur_Binnen
Buitenmuur_Na.Temperature_Outside = Temperatuur_Buiten
Buitenmuur_Na.Add ( Rsi )
Buitenmuur_Na.Add ( Kalkzandsteen )
Buitenmuur_Na.Add ( Glaswol_5cm )
Buitenmuur_Na.Add ( Iso_Parels_5cm )
Buitenmuur_Na.Add ( Baksteen )
Buitenmuur_Na.Add ( Rse )
print ( Buitenmuur_Na )

Name        = Buitenmuur Na Bijvulling
Lengte      = 9.4 [m]
Hoogte      = 2.5 [m]
Oppervlakte = 23.5 [m2]
R-Waarde    = 3.25 [m2.K/W]
U-Waarde    = 0.31 [W/m2.K]
T-Inside    = 20.0 [Celcius]
T-Outside   = 0.0 [Celcius]
Heat Loss   = 144 [W]
===========  Kosten per m2, 24 uur per dag, 100 dagen =========
Heat Loss 1m2   = 6 [W]
Energy Cost 1m2 = 1.18 [Euro]
Gasprijs        = 0.80 [Euro]
Layer[s]    = [ Rsi, Kalkzandsteen, Glaswol 5cm, Iso-Parels 5cm, Baksteen, Rse,  ]
Dikte[s]    = [ -1.000, -1.000, 0.050, 0.050, -1.000, -1.000,  ] [m]
R-Waarde[s] = [ 0.13, 0.10, 1.43, 1.47, 0.08, 0.04,  ] [m2.K/W]
U-Waarde[s] = [ 7.69, 10.00, 0.70, 0.68, 12.50, 25.00,  ] [W/m2.K]
T-Inside    = 20.0 [Celcius]
T-Outside   = 0.0 [Celcius]



## Muur met half gevulde spouw + Binnenwand

In [12]:
Buitenmuur_Super = Sandwich ( "Buitenmuur Super" )
Buitenmuur_Super.Inpandig            = False
Buitenmuur_Super.Lengte              = Lengte
Buitenmuur_Super.Hoogte              = Hoogte
Buitenmuur_Super.Temperature_Inside  = Temperatuur_Binnen
Buitenmuur_Super.Temperature_Outside = Temperatuur_Buiten
Buitenmuur_Super.Add ( Rsi )
Buitenmuur_Super.Add ( Gipsplaat )
Buitenmuur_Super.Add ( PIR_alu_10cm )
Buitenmuur_Super.Add ( Kalkzandsteen )
Buitenmuur_Super.Add ( Glaswol_5cm )
Buitenmuur_Super.Add ( Spouw_Zwak_Geventileerd )
Buitenmuur_Super.Add ( Baksteen )
Buitenmuur_Super.Add ( Rse )

print ( Buitenmuur_Super )

Name        = Buitenmuur Super
Lengte      = 9.4 [m]
Hoogte      = 2.5 [m]
Oppervlakte = 23.5 [m2]
R-Waarde    = 6.54 [m2.K/W]
U-Waarde    = 0.15 [W/m2.K]
T-Inside    = 20.0 [Celcius]
T-Outside   = 0.0 [Celcius]
Heat Loss   = 71 [W]
===========  Kosten per m2, 24 uur per dag, 100 dagen =========
Heat Loss 1m2   = 3 [W]
Energy Cost 1m2 = 0.59 [Euro]
Gasprijs        = 0.80 [Euro]
Layer[s]    = [ Rsi, Gipsplaat, PIR_alu_10cm, Kalkzandsteen, Glaswol 5cm, Spouw Zwak Geventileerd, Baksteen, Rse,  ]
Dikte[s]    = [ -1.000, -1.000, -1.000, -1.000, 0.050, -1.000, -1.000, -1.000,  ] [m]
R-Waarde[s] = [ 0.13, 0.04, 4.54, 0.10, 1.43, 0.18, 0.08, 0.04,  ] [m2.K/W]
U-Waarde[s] = [ 7.69, 25.00, 0.22, 10.00, 0.70, 5.56, 12.50, 25.00,  ] [W/m2.K]
T-Inside    = 20.0 [Celcius]
T-Outside   = 0.0 [Celcius]



## Muur met lege spouw

In [13]:
Buitenmuur_Leeg = Sandwich ( "Buitenmuur Leeg" )
Buitenmuur_Leeg.Inpandig            = False
Buitenmuur_Leeg.Lengte              = Lengte
Buitenmuur_Leeg.Hoogte              = Hoogte
Buitenmuur_Leeg.Temperature_Inside  = Temperatuur_Binnen
Buitenmuur_Leeg.Temperature_Outside = Temperatuur_Buiten
Buitenmuur_Leeg.Add ( Rsi )
Buitenmuur_Leeg.Add ( Kalkzandsteen )
Buitenmuur_Leeg.Add ( Spouw_Zwak_Geventileerd )
Buitenmuur_Leeg.Add ( Baksteen )
Buitenmuur_Leeg.Add ( Rse )

print ( Buitenmuur_Leeg )

Name        = Buitenmuur Leeg
Lengte      = 9.4 [m]
Hoogte      = 2.5 [m]
Oppervlakte = 23.5 [m2]
R-Waarde    = 0.53 [m2.K/W]
U-Waarde    = 1.89 [W/m2.K]
T-Inside    = 20.0 [Celcius]
T-Outside   = 0.0 [Celcius]
Heat Loss   = 886 [W]
===========  Kosten per m2, 24 uur per dag, 100 dagen =========
Heat Loss 1m2   = 37 [W]
Energy Cost 1m2 = 7.25 [Euro]
Gasprijs        = 0.80 [Euro]
Layer[s]    = [ Rsi, Kalkzandsteen, Spouw Zwak Geventileerd, Baksteen, Rse,  ]
Dikte[s]    = [ -1.000, -1.000, -1.000, -1.000, -1.000,  ] [m]
R-Waarde[s] = [ 0.13, 0.10, 0.18, 0.08, 0.04,  ] [m2.K/W]
U-Waarde[s] = [ 7.69, 10.00, 5.56, 12.50, 25.00,  ] [W/m2.K]
T-Inside    = 20.0 [Celcius]
T-Outside   = 0.0 [Celcius]



## Vergelijk van de verschillende spouwmuren
Deze tabel functie is gemaakt om het warmteverlies van een gehele ruimte of woning in detail weer te geven. In de huidige situatie waar een aantal spouwmuur constructies met elkaar worden vergeleken, is de onderste regel "Totaal", evenals enkele kolommen, volkomen zinloos. 
<br>Het rechter deel van de tabel (kolommen met de getallen erboven) geeft de kolomnaam de buitentemperatuur aan en de waarden in die kolom staan, is het warmteverlies in Watt dat door de muur weglekt bij die buitentemperatuur. 
<br>kolom Kosten    = de geschatte totale jaarlijke Energiekosten in Euros (bij gebruik van Gas)
<br>kolom Kosten-m2 = de geschatte jaarlijke Energiekosten per m2 in Euros (bij gebruik van Gas)

In [14]:
Ts  = range ( -8, 12, 2 )
Elements = [ Buitenmuur_Leeg, Buitenmuur_Nu, Buitenmuur_Na, Buitenmuur_Super ]
Heat_Transfer_Table ( Elements, Ts )

['Buitenmuur Leeg', 'Buitenmuur Nu', 'Buitenmuur Na Bijvulling', 'Buitenmuur Super']


R-waarde  U-waarde  Oppervlak  T-Binnen  Inpandig  \
Buitenmuur Leeg               0.53      1.88       23.5        20     False   
Buitenmuur Nu                 1.95      0.51       23.5        20     False   
Buitenmuur Na Bijvulling      3.24       0.3       23.5        20     False   
Buitenmuur Super              6.53      0.15       23.5        20     False   
Totaal                    R-waarde  U-waarde  Oppervlak  T-Binnen  Inpandig   

                          Kosten-m2  Kosten    -8    -6    -4    -2     0  \
Buitenmuur Leeg                7.24     170  1241  1152  1064   975   886   
Buitenmuur Nu                  1.96      46   335   311   287   263   239   
Buitenmuur Na Bijvulling       1.18      27   202   188   173   159   144   
Buitenmuur Super               0.58      13   100    93    86    79    71   
Totaal                    Kosten-m2  Kosten  1878  1744  1610  1476  1340   

                             2     4    6    8   10  
Buitenmuur Leeg            798   709  620  532  443  
Buitenmuur Nu              215   191  167  143  119  
Buitenmuur Na Bijvulling   130   115  101   86   72  
Buitenmuur Super            64    57   50   43   35  
Totaal                    1207  1072  938  804  669

## Conclusie
Als ik bovenstaande tabel zie, krijg ik een vermoeden waarom isolatiebedrijven alleen geïnteresseerd zijn in het vullen van een geheel lege spouw, namelijk de besparing bedraagt ongeveer 6 Euro per m2 per jaar en dus kun je het goed verkopen dat een m2 buitenmuur isoleren 15 tot 20 Euro kost. De claim van de isolatiebedrijven over de terugverdientijd lijkt hier dus te kloppen.
<br><br>Voor een half gevulde spouw, bedraagt de economische winst "slechts" 0.80 Euro, hetgeen betekent dat er een terugverdientijd van ongeveer 20 jaar aan vasthangt.
<br><br>De supermuur, met een extra inpandige voorzetwand, bespaart t.o.v. de half gevulde spouw 1.40 Euro. Als ik een prijs gok van 25 Euro per m2, zou dat een terugverdientijd van ongeveer 15 jaar betekenen (dit allemaal uit het hoofd). Energetisch gezien is dit een hele fraaie oplossing, immers de hier gerealiseerde Rc waarde (6.5 m2.K/W) voldoet zelfs ruim aan de norm voor nieuwbouw gevels (4.5 m2.K/W)